kuber/kube.go

```diff
@@ -501,6 +501,20 @@ func (kube *Kube) GetPods() (*kv1.PodList, error) {
 	return podList, nil
 }
 
+// GetPods get kubernetes pods for namespace
+func (kube *Kube) GetNameSpacePods(namespace string) (*kv1.PodList, error) {
+	kube.logger.Debugf(nil, "{kubernetes} retrieving list of pods")
+	podList, err := kube.core.Pods(namespace).List(kmeta.ListOptions{})
```

Comment:
Would you discuss with the devops the ability of using the `listOptions` to filter the pods from the api server using the labels of the fields of the deployment instead of looping through each pod.


the same goes for the other queries we made.

@mohamedveron

Reply:
ok

Reply:
not all the deployments have labels so it is the most suitable option for now